tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -73,36 +73,23 @@ jobs:
 - job: Update_Dashboard
   workspace:
     clean: all
-  pool:
-    vmImage: 'ubuntu-latest'
+  variables:
+  - name: skipComponentGovernanceDetection
+    value: true
+  pool: 'Linux-CPU-2019'
   dependsOn: Android_CI
-  # disable update dashboard of Android Code coverage until Azure CI machine cannot access database issue is resolved
-  # condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI'))
```

Comment:
Do you want to enable this line again? Or is there any way we can tell the Build.Reason in the records?

Reply:
It's ok to either have it or not. Because the data has a 'branch' name and we can filter it out when presenting the dashboard.